convene-web/app/models/room.rb

```diff
@@ -12,7 +12,7 @@ class Room < ApplicationRecord
 
   # FriendlyId's does the legwork to make the slug uri-friendly
   extend FriendlyId
-  friendly_id :name, use: :slugged
+  friendly_id :name, use: :scoped, scope: :workspace
```

Comment:
Ref: http://norman.github.io/friendly_id/file.Guide.html#Unique_Slugs_by_Scope

Reply:
Nice call. Perhaps adding a comment near the implementation or test linking to the docs would provide a breadcrumb for folks to follow in the future without having to immediately go through search.